core/src/test/java/greencity/controller/EventCommentControllerTest.java

```diff
@@ -0,0 +1,118 @@
+package greencity.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import greencity.config.SecurityConfig;
+import greencity.converters.UserArgumentResolver;
+import greencity.dto.eventcomment.AddEventCommentDtoRequest;
+import greencity.dto.user.UserVO;
+import greencity.service.EventCommentService;
+import greencity.service.UserService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+import org.modelmapper.ModelMapper;
+import org.springframework.context.annotation.Import;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableHandlerMethodArgumentResolver;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import java.security.Principal;
+
+import static greencity.ModelUtils.getPrincipal;
+import static greencity.ModelUtils.getUserVO;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+@ContextConfiguration
+@Import(SecurityConfig.class)
+class EventCommentControllerTest {
+    private static final String eventCommentControllerLink = "/events/comments";
+    private MockMvc mockMvc;
+    @InjectMocks
+    private EventCommentController eventCommentController;
+    @Mock
+    private EventCommentService eventCommentService;
+    @Mock
+    private UserService userService;
+    @Mock
+    private ModelMapper modelMapper;
+    private final Principal principal = getPrincipal();
+
+    @BeforeEach
+    void setup() {
+        this.mockMvc = MockMvcBuilders.standaloneSetup(eventCommentController)
+            .setCustomArgumentResolvers(new PageableHandlerMethodArgumentResolver(),
+                new UserArgumentResolver(userService, modelMapper))
+            .build();
+    }
+
+    @Test
+    void save() throws Exception {
```

Comment:
You can @SneakyThrows annotation here and in the rest of the tests, no need to have this exception